components/SponsorItem.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box, BoxProps, Image, Link } from "@chakra-ui/core";
+import styled from "@emotion/styled";
+import { ReactElement } from "react";
+
+export interface SponsorProps extends BoxProps {
+  imgUrl: string;
+  sponsorUrl: string;
+  faded?: boolean;
+}
+
+const SponsorItem = styled(SponsorItemInner)`
+  margin: 32px 0;
+  transition: all 0.3s ease-in-out;
+  transition-delay: 0.1s
+`;
+
+const SponsorLogo = styled(Image)`
+  user-select: none;
+  margin: 0 auto;
+`
+
+function SponsorItemInner({imgUrl, sponsorUrl, faded, ...boxProps}: SponsorProps): ReactElement {
+  return <Box {...boxProps} opacity={faded ? 0.3 : 1}>
+    <Link isExternal href={sponsorUrl}>
```

Comment:
Can you add `display="block" boxShadow="none !important"` to Link here?

When the link is in focus it has a partially shown box-shadow highlight:

![image](https://user-images.githubusercontent.com/20251243/102020561-fb1ebc80-3d47-11eb-97cb-76a359e1737a.png)

It's good for accessibility but personally I don't think it looks great here :sweat_smile: